md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/DQSetupAPI.java

```diff
@@ -51,19 +52,18 @@
     DataQualityDAO dataQualityDAO;
 
     @RequestMapping(value = "/{id}", method = RequestMethod.DELETE)
-    public
-    @ResponseBody
+    @ResponseBody public
     RestWrapper delete(@PathVariable("id") Integer processId, Principal principal) {
         RestWrapper restWrapper = null;
         try {
             DQSetupInfo dqSetupInfo = new DQSetupInfo();
             dqSetupInfo.setSubProcessId(processId);
-            //s.delete("call_procedures.DeleteDQSetup",dqSetupInfo);
             dataQualityDAO.deleteDQSetup(processId);
             restWrapper = new RestWrapper(null, RestWrapper.OK);
-            LOGGER.info("Record with ID:" + processId + " deleted from DQSetup by User:" + principal.getName());
+            LOGGER.info(RECORDWITHID + processId + " deleted from DQSetup by User:" + principal.getName());
         } catch (Exception e) {
-            restWrapper = new RestWrapper(e.getMessage(), RestWrapper.ERROR);
+            LOGGER.error("error occurred while uploading file", e);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "error occurred while uploading file" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1192)